### PR TITLE
TK Django exercise - a simplistic recipe DB

### PR DESCRIPTION
**Pull request by clicktravel-rob**
The migrations directory wasn't on the default `.gitignore` for python, so I included it.
I suspect you wouldn't normally include the migrations, since it's running on Docker.

### CODE REVIEW

**Comment by israel-tk** [comment]
> The migrations directory wasn't on the default `.gitignore` for python, so I included it.
> I suspect you wouldn't normally include the migrations, since it's running on Docker.

Django migrations should be included in the repo even if it is running on Docker. Why do you think they shouldn't?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: python
```
Comment (israel-tk) [inline .travis.yml:1]: No need to configure this to run on travis, but OK if you want so. We use CircleCI as Continuous Integration system though. :)

**Review by israel-tk — APPROVED** [review]
All good! Nice job!

File: .travis.yml (hunk 1 above)
Comment (clicktravel-rob) [reply .travis.yml:1]: Cool - it was jut that the Udemy course used it.

**Comment by clicktravel-rob** [comment]
> > The migrations directory wasn't on the default `.gitignore` for python, so I included it.
> > I suspect you wouldn't normally include the migrations, since it's running on Docker.
> 
> Django migrations should be included in the repo even if it is running on Docker. Why do you think they shouldn't?

Since they're auto-generated - as a rule-of-thumb, anything that can be regenerated could be excluded.
But it depends on what is getting deployed, I guess.

**Pull request merged**